tests/test_engine.py

```diff
@@ -11,17 +12,18 @@ class FakeTask:
 @pytest.mark.usefixtures("engine")
 def test_check_argument(engine):
     with pytest.raises(InvalidArgumentError):
-        engine.execute([FakeTask])
```

Comment:
Adding a class here is always a problem, so I'm trying to add an instance instead (that fails as well but for a more common reason).